docs/components/Chips/Web.stories.tsx

```diff
@@ -97,6 +88,7 @@ const SelectionTemplate: ComponentStory<typeof Chips> = args => {
   return (
     <Chips
       {...args}
+      type="dismissible"
```

Comment:
seems like we're doing this same prop assignment twice

```
export const Selection = SelectionTemplate.bind({});
Selection.args = {
  type: "dismissible",
};
```

so `{...args}` is doing `type="dismissible"` but since it's defined as `ChipProps` it _could_ be dismissible which is not enough to satisfy the XOR-ish behavior

I'm not entirely sure why we do this whole `bind` pattern. it seems to upset a bunch of types. regardless, I guess this is fine. at worst it's a bit redundant.